src/host/screenInfo.cpp

```diff
@@ -2035,10 +2035,15 @@ const SCREEN_INFORMATION& SCREEN_INFORMATION::GetMainBuffer() const
 
     const FontInfo& existingFont = GetCurrentFont();
 
+    // The buffer needs to be initialized with the standard erase attributes,
+    // i.e. the current background color, but with no meta attributes set.
+    auto initAttributes = GetAttributes();
+    initAttributes.SetMetaAttributes(0);

```

Comment:
(this guy, and other places in this file, need `ExtendedAttributes` reset too)

Reply:
In case it's not clear, this has been updated (as have all of the other places where the meta attributes are reset). For some reason GitHub hasn't detected that these snippets are all now outdated.